### PR TITLE
fix bug, add comments (field package)

### LINKED ISSUE #39: Describe serialization format for map
User should be able to load map from file. We need to create convenient format to describe the map.

Example:
```5 7
.......
...#...
...#...
...#...
P.###.Z
```

In this example file contains two numbers: `n` and `m`, which denote amount of rows and columns of a map respectively. `.` is an empty space, `#` is a wall, `Z` is zombie and `P` is player spawn position

### PR DESCRIPTION
**Pull request by ZhekehZ**

### CODE REVIEW

File: src/main/java/ru/itmo/roguelike/field/Chunk.java [hunk 1]
```diff
@@ -10,12 +14,18 @@
     public static final int HEIGHT_IN_PIX = WIDTH_IN_TILES * Tile.HEIGHT_IN_PIX;
 
     private static final float[][] chunkValues = new float[WIDTH_IN_TILES][HEIGHT_IN_TILES];
-    // Single thread only!
+    // "Single thread only" cause
```
Comment (HarmlessEvil) [inline src/main/java/ru/itmo/roguelike/field/Chunk.java]: Fields may also be documented with javadoc. I think, it'd be better to add javadoc to this field, including this comment

File: src/main/java/ru/itmo/roguelike/field/Chunk.java [hunk 2]
```diff
@@ -29,12 +39,18 @@ public Chunk(int x, int y, NoiseGenerator generator, MobPositionGenerator mobGen
         reInitTiles(x, y, generator);
     }
 
+    /**
+     * @return tile at world-coordinate (x, y)
```
Comment (HarmlessEvil) [inline src/main/java/ru/itmo/roguelike/field/Chunk.java]: Let's stick to format `{@code <x, y>}` (everyone has his own notation, let's keep in similar)

File: src/main/java/ru/itmo/roguelike/field/Field.java [hunk 3]
```diff
@@ -6,16 +6,37 @@
 import javax.swing.text.html.Option;
 import java.util.Optional;
 
+/**
+ *  Game field (map)
```
Comment (HarmlessEvil) [inline src/main/java/ru/itmo/roguelike/field/Field.java]: ```The main game field. It represents a world man where all objects are located. It divided to tiles ({@link Tile}), where all tiles may be free or solid. Actors ({@link Actor}) may move only on free tiles```

File: src/main/java/ru/itmo/roguelike/field/Chunk.java (hunk 1 above)
Comment (ZhekehZ) [reply src/main/java/ru/itmo/roguelike/field/Chunk.java]: Это же private поле, оно все равно не будет выкинуто при компиляции документации

**Pull request merged**